addons/account/tests/test_account_move_date_algorithm.py

```diff
@@ -49,7 +49,7 @@ def _reverse_invoice(self, invoice):
             .create({
                 'journal_id': invoice.journal_id.id,
                 'reason': "no reason",
-                'refund_method': 'cancel',
+                'refund_method': 'refund',
```

Comment:
Shouldn't it be 'modify'?

Reply:
the test just checks the date, 'modify' would change the behaviour